services/api/src/admin_new_email_notification.py

```diff
@@ -104,25 +112,32 @@ def options(self):
         # CORS support
         return ("", 204, self.options_headers)
 
+    @require_permission(required_role=None)
     def get(self):
         logging.debug("AdminNewNotification GET requested")
+
         # Check for main body values
         schema = AdminGetNotificationSchema()
         errors = schema.validate(request.args)
         if errors:
             logging.error(str(errors))
             abort(400, str(errors))
         user_email = urllib.request.unquote(request.args["user_email"])
```

Comment:
question: is unquote needed here? Flask should be handling the url parameter decode automatically for us

Reply:
This is indeed required